docs/package.json

```diff
@@ -9,12 +9,20 @@
     "help": "docusaurus --help",
     "swizzle": "docusaurus swizzle",
     "deploy": "docusaurus deploy",
+    "write-translations": "docusaurus write-translations",
+    "write-heading-ids": "docusaurus write-heading-ids",
+    "write-i18n": "npm run write-heading-ids && npm run write-translations",
     "format": "prettier --check \"**/*.{js,jsx,json,md,scss}\"",
     "format:fix": "prettier --write \"**/*.{js,jsx,json,md,scss}\"",
     "lint": "set -ex; eslint .",
-    "lint:fix": "npm run lint -- --fix"
+    "lint:fix": "npm run lint -- --fix",
+    "crowdin": "crowdin",
```

Comment:
Devs can use `npx crowdin ...` and the scripts can use `crowdin ...` directly, I don't think this shortcut is needed

Reply:
You're probably right, will just add it to the docs.

Reply:
Added a Translation paragraph to the README.